docs/workflow-archive.md

```diff
@@ -2,7 +2,7 @@
 
 > v2.5 and after
 
-If you want to keep completed workflows for a long time, you can use the workflow archive to save them in a Postgres or MySQL (>= 5.7.8) database.
+If you want to keep completed workflows for a long time, you can use the workflow archive to save them in a Postgres (>=9.4) or MySQL (>= 5.7.8) database.
```

Comment:
Thanks for adding this 👍 . I mentioned this part of the docs in https://github.com/argoproj/argo-workflows/issues/13295#issuecomment-2218348570 et al too

Reply:
It might be good to add an optional `Job` to the Helm Charts or within the CLI or manifests that tests for compatibility before a deployment -- i.e. min postgres, mysql, and k8s versions as all external deps